src/Place/Canonical/DuplicatePlaceRepository.php

```diff
@@ -7,18 +7,20 @@
 interface DuplicatePlaceRepository
 {
     /**
-     * @return int[]
+     * @return string[]
      */
     public function getClusterIds(): array;
 
     /**
      * @return string[]
      */
-    public function getPlacesInCluster(int $clusterId): array;
+    public function getPlacesInCluster(string $clusterId): array;
 
-    public function setCanonicalOnCluster(int $clusterId, string $canonical): void;
+    public function setCanonicalOnCluster(string $clusterId, string $canonical): void;
 
     public function getCanonicalOfPlace(string $placeId): ?string;
 
     public function getDuplicatesOfPlace(string $placeId): ?array;
+
+    public function markAsProcessed(string $placeId): void;
```

Comment:
The new function does not have a unit test.

Reply:
You ask, I provide...